resources/views/emails/check-in-notification.blade.php

```diff
@@ -0,0 +1,6 @@
+<h1>From: {{ $phone_number->value }}</h1>
```

Comment:
lol

Reply:
You *should* be able to just put `$phone_number` in there and it'll stringify itself.

[AppServiceProvider.php:20](https://github.com/inxilpro/disastercheckin.com/blob/efd3084947c8f8a800b80f4868157109240ac89d/app/Providers/AppServiceProvider.php#L20C4-L20C121)